### ISSUE TITLE
Timeline to 4.6.3?

### ISSUE THREAD

**Issue by eightbitraptor** [seattlerb/flog#67]
I'm running into an issue when running `flog` `4.6.2` in a Rails application containing the following code:

```ruby
respond_with({}, status: :ok, location: false) do |format|
  format.html { redirect_to some_path, notice: "some notice" }
end
```

`flog` fails with the following error

```
/Users/mattvh/.gem/ruby/2.5.5/gems/flog-4.6.2/lib/flog.rb:151:in `dsl_name?': undefined method `[]' for nil:NilClass (NoMethodError)
```

After some investigation, I have ascertained that the `dsl_name` method assumes that when a hash is passed as the first argument to a method, it will never be an empty hash literal, and it attempts to index various parts out of the resulting `sexp` to determine what structure the hash has.

I have fixed this in this commit: https://github.com/eightbitraptor/flog/commit/c97dd0cbe73d218f382e8f455e5ea14afde0adcb

I have also ported my test over to the `master` branch, where I think it's already been fixed. https://github.com/eightbitraptor/flog/commit/ea7957ed7aa10cf75507daafa23a9c1cdc5686fd

It's not obvious to me what the intended Hoe workflow is for working against a patched library version inside a project managed by Bundler. Currently I'm doing the following things:

* vendoring `flog` into our app with `bundle unpack flog`
* using `hoe-gemspec` to generate a pre-release gemspec
* pointing our `Gemfile` at the pre release package in `vendor/gems`

I'd like to be able to just use upstream code directly, but in order to do that I'd need 4.6.3 to be released - assuming that my supposition is correct and this behaviour is suitably fixed in `master`

Is there any plan in the near future to release `4.6.3`? Thanks

**Comment by zenspider** [comment]
Send me a PR and I'll get it out as soon as I merge / validate.

**Comment by eightbitraptor** [comment]
Thank you @zenspider 👍 let me know if you need anything else from me!

**Comment by zenspider** [comment]
I overlooked the PR, but I just released 4.6.3.

The PR will make it in to the next release. Sorry about that.
